src/screens/konnectors/LauncherView.js

```diff
@@ -259,7 +261,10 @@ class LauncherView extends Component {
               <WebView
                 key={this.state.workerKey}
                 mediaPlaybackRequiresUserAction={true}
-                ref={ref => (this.workerWebview = ref)}
+                ref={ref => {
+                  this.workerWebview = ref
+                  this.launcher.emit('worker:webview:ready')
+                }}
```

Comment:
I can understand this change, but I'm not understanding the change made here https://github.com/cozy/cozy-flagship-app/pull/777/commits/69410ca7c301e72e034dd89abcde1db416c9e735#diff-83374be02352a4c209865e6cab112ff6d487b98a8e9000cc58e5e42e7898a993R173-R174

Can you explain it please? 

Also, you should test that `ref` is not null. Ref callback can be called with ref === null and I think that in that case you should not emit. See https://github.com/facebook/react/issues/4533 & https://github.com/facebook/react/pull/8333/files

Reply:
This other event emit is done when the url change does not need any webview reload. In this case, we know that the webview reference won't change and we can resolve the `setWorkerState` promise directly

Reply:
Maybe adding this in a comment can be useful? 🙏